README.md

```diff
@@ -44,7 +44,8 @@ No further dependencies other than those defined in `go.mod`
 | DEFAULT_OFFSET                              | 0                               | The default offset of search results                                                                               |
 | DEFAULT_PAGE                                | 1                               | The default current page of search results                                                                         |
 | DEFAULT_SORT                                | relevance                       | The default sort of search results                                                                                 |
-| ENABLE_REWORKED_DATA_AGGREGATION_PAGES      | false                           | Enable the reworked data aggregation pages                                                                         |
+| ENABLE_AGGREGATION_PAGES                    | false                           | Enable the aggregation pages, is a combination feature flag with ENABLE_TOPIC_AGGREGATION_PAGES                    |
+| ENABLE_TOPIC_AGGREGATION_PAGES              | false                           | Enable the topic aggregation pages, is a combination feature flag with ENABLE_AGGREGATION_PAGES                    |
```

Comment:
So just be clear here that to enable the topic ones you have to enable the aggregation pages as well.